docs/source/classifier.rst

```diff
@@ -1,5 +1,3 @@
-
-===================================
 The MRIQC classifier for T1w images
 ===================================
 
```

Comment:
```suggestion
```

If we are going to curate all the docs this time, please remove unnecessary spaces after headings :)